packages/layout/src/tree-layout/layout-component/dummy-component.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable require-jsdoc */
+
+/* eslint no-undef: "off" */
+
+import LayoutComponent from './layoutComponent';
+
+class DummyComponent extends LayoutComponent {
+    constructor (seed, dimensions) {
+        super(seed, dimensions);
+        this.seed = seed;
+    }
+
+    getLogicalSpace () {
+        return {
+            width: this.boundBox.width - (2 * this.seed),
+            height: this.boundBox.height - (2 * this.seed)
+        };
+    }
+
+    setSpatialConfig (conf) {
+        this.boundBox.top = conf.y;
```

Comment:
Have a method that acts as a getter and setter to change values of instance variables